### ISSUE TITLE
Blessing of Sanctuary

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#2827]
**Spell ID's:** 20914

> not working at all

**Comment by Shadownflame** [comment]
1) The damage reduction effect doesn't work. 
2) The reflect damage procs from parries too, not only blocks

**Comment by ghost** [comment]
I'm not able to replicate this. The damage reduction is working fine, and so is the reflect. This spell applies the DR before other calculations, most notably before armor. If you get hit for 300 damage, it then reduces it by 24, and then armor kicks in and reduces it. I have 53.98% armor right now so it'd end up being a hit that did 148 damage, without Sanctuary it'd have hit for 161 damage. So it only reduced it by 13.

As for the reflect, I couldn't get it to proc from any parries, it consistently procced from blocks though.

**Comment by ghost** [comment]
Feel free to reopen if you were able to get this to happen again though

**Comment by Shadownflame** [comment]
Yep the damage reduction works fine it just applies before armor and that makes it so weak it gets lost in the regular min-max range of the mob attacks. Works as intended. *sigh*

The parry issue is consistently reproducible though, and you get the damage proc from Dodges too. Requires the use of Greater Blessing of Sanctuary (ID 25899) not the regular one. Somehow the 10min one works properly.

**Comment by ghost** [comment]
Ah ha you're right, and I see the problem. Thanks, didn't think to check greater
